client-ts/Microsoft.AspNetCore.SignalR.Test.Server/wwwroot/js/hubConnectionTests.js

```diff
@@ -258,13 +258,13 @@ describe('hubConnection', function () {
                 hubConnection.start().then(function () {
                     return hubConnection.invoke('InvokeWithString', message);
                 })
-                .then(function() {
-                    return hubConnection.stop();
-                })
-                .catch(function (e) {
-                    fail(e);
-                    done();
-                });
+                    .then(function () {
```

Comment:
I liked the old indentation

Reply:
That's the VSCode auto-formatter at work :(.